plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -75,6 +77,13 @@ func (l *persistentVolumeLabel) Admit(a admission.Attributes) (err error) {
 		}
 		volumeLabels = labels
 	}
+	if volume.Spec.GCEPersistentDisk != nil {
+		labels, err := l.findGCEPDLabels(volume)
+		if err != nil {
+			return admission.NewForbidden(a, fmt.Errorf("error querying GCE PD volume %s: %v", volume.Spec.GCEPersistentDisk.PDName, err))
```

Comment:
This broke provisioning (because we don't have an e2e test that would have prevented this change).

The provisioner creates a dummy PV in response to a claim that requires provisioning.  The "dummy" ID on the struct was required to pass validation. A controller reconciles this PV with the infrastructure and creates a resource asynchronously.

There is no "dummy" PD in the provider yet, so we can't look it up for labels.  This admission controller makes it required that the PD exist.  It could be, too, that making a placeholder PV is wrong.  I am open to suggestions.